types/yargs/index.d.ts

```diff
@@ -427,7 +427,10 @@ declare namespace yargs {
         parsed: DetailedArguments | false;
 
         /** Allows to configure advanced yargs features. */
-        parserConfiguration(configuration: Partial<Configuration>): Argv<T>;
+        parserConfiguration(configuration: Partial<Configuration & {
+            /** Sort commands alphabetically. Default is `false` */
+            'sort-commands': boolean;
```

Comment:
Intersection type in the generic type param is kind of gross.  Can we pull it out and name it to clean this up?

Reply:
Sure thing. I've pulled it out to a defined type instead of being inline. Thanks for the feedback.